core/modules/views/includes/view.inc

```diff
@@ -1992,6 +1992,12 @@ class view {
 
   /**
    * Find and initialize the localization plugin.
+   *
+   * @return bool
+   *   Whether the active plugin does handle translation or not. FALSE for the
+   *   the plugin "none", or if no plugin was available.
```

Comment:
Remove extra "the" (sorry I missed that in my initial review)

```suggestion
   *   plugin "none", or if no plugin was available.
```

Reply:
Oops, right, the the...